idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -3,7 +3,7 @@
 -- TODO? replace all 'addr bytea' with 'addr_id bigint' and a mapping table? makes addrs an 8 byte int that fits in a register instead of a 32 byte string
 
 CREATE TABLE IF NOT EXISTS block_header (
-  round bigint PRIMARY KEY,
+  round numeric(20) PRIMARY KEY,
```

Comment:
What is the rationale behind choosing 20 as opposed to another number?

Reply:
this is because uint64 max is 18446744073709551615 and it's a value of 20 digits.

Reply:
Got it thanks!